core/main/src/firebolt/handlers/privacy_rpc.rs

```diff
@@ -549,9 +553,22 @@ impl PrivacyImpl {
                     "PrivacySettingsStorageType::Cloud: Not Available",
                 )))
             }
-            // PrivacySettingsStorageType::Sync => Err(jsonrpsee::core::Error::Custom(String::from(
-            //     "PrivacySettingsStorageType::Sync: Unimplemented",
-            // ))),
+            PrivacySettingsStorageType::Sync => {
+                let dist_session = platform_state.session_state.get_account_session().unwrap();
```

Comment:
This unwrap is not guaranteed to be safe always. Can we please check if it is none?